tests/code-check-roave-backward-compatibility-check-on-pr/GITHUB_EVENT_CONTENTS.json

```diff
@@ -0,0 +1,328 @@
+{
```

Comment:
A github event is **HUGE** :O

Reply:
Do we really need the whole event here? is there a check which verifies that the JSON is 100% available?

I would expect that we just pass those parts to the matrix which are actually used. Makes things easier as this event might grow in the future anyways and thus we will never have 100% available.

Reply:
This is the correct event according to its JSON Schema: I wouldn't cut a corner here. 

 > I would expect that we just pass those parts to the matrix which are actually used. 

The structure is based on the octokit type (and associated type cast).

 > Makes things easier as this event might grow in the future anyways and thus we will never have 100% available.

Would rather say that we get a BC break warning heads-up, should they expand it further :D 

So far, new fields are marked optional: I also skipped all optional fields from here.